src/CommonLib/Processors/LDAPPropertyProcessor.cs

```diff
@@ -761,4 +913,19 @@ public class ComputerProperties
         public TypedPrincipal[] SidHistory { get; set; } = Array.Empty<TypedPrincipal>();
         public TypedPrincipal[] DumpSMSAPassword { get; set; } = Array.Empty<TypedPrincipal>();
     }
-}
\ No newline at end of file
+    
+    /// <summary>
+    /// Holds list of SID history and principles.
+    /// </summary>
+    public class SidHistory
```

Comment:
I'm exposing what should be a private class here in the name of testability.  This is a consequence of `ReadSidHistory` above.

Reply:
This has been rightfully removed.